### ISSUE TITLE
Reset password feature

### ISSUE THREAD

**Issue by matnel** [HIIT/dime-server#51]

**Comment by mvsjober** [comment]
BTW, any ideas how to reset your password without an email being available in the system? :-)

**Comment by matnel** [comment]
For local installations, just run a script that changes it and is run locally?

**Comment by mvsjober** [comment]
Yes, I guess that's the only solution. I can't be done in the normal DiMe UI.

**Comment by mvsjober** [comment]
Note to self, this might be the easiest way to do it: <http://nixmash.com/java/using-spring-boot-commandlinerunner/>
